modules/transfer.py

```diff
@@ -10,14 +10,15 @@
 
 
 class Transfer:
-    def __init__(self, config: Config):
+    def __init__(self, config: Config, pubkey: Path):
```

Comment:
We have some work to do ensure make cosign signature validation optional, and provide a way to determine what registries/projects are validated, so this could be optional.  But that's probably a different issue altogether.

Reply:
I agree.  Plus, to make it even more complicated, some people may a have variety of registries that they would like to verify and each with a separate key.